packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -1,9 +1,66 @@
 import { LocaleText } from '../models/api/localeTextApi';
 
 export const DEFAULT_LOCALE_TEXT: LocaleText = {
+  // Root
+  rootGridLabel: 'grid',
+
+  // Density selector toolbar button text
   toolbarDensity: 'Density',
   toolbarDensityLabel: 'Density',
   toolbarDensityCompact: 'Compact',
   toolbarDensityStandard: 'Standard',
   toolbarDensityComfortable: 'Comfortable',
+
+  // Columns selector toolbar button text
+  toolbarColumns: 'Columns',
+  toolbarColumnsLabel: 'Show Column Selector',
+
+  // Filters toolbar button text
+  toolbarFilters: 'Filters',
+  toolbarFiltersLabel: 'Show Filters',
+  toolbarFiltersTooltipHide: 'Hide Filters',
+  toolbarFiltersTooltipShow: 'Show Filters',
+  toolbarFiltersTooltipActive: (count) => `${count} active filter(s)`,
+
+  // Columns panel text
+  columnsPanelTextFieldLabel: 'Find column',
+  columnsPanelTextFieldPlaceholder: 'Column title',
+  columnsPanelDragIconLabel: 'Reorder Column',
+  columnsPanelShowAllButton: 'Show All',
+  columnsPanelHideAllButton: 'Hide All',
+
+  // Filter panel text
+  filterPanelAddFilter: 'Add Filter',
+  filterPanelDeleteIconLabel: 'Delete',
+  filterPanelOperators: 'Operators',
+  filterPanelOperatorAnd: 'And',
+  filterPanelOperatorOr: 'Or',
+  filterPanelColumns: 'Columns',
+
+  // Column menu text
+  columnMenuLabel: 'Menu',
+  columnMenuShowColumns: 'Show columns',
+  columnMenuFilter: 'Filter',
+  columnMenuHideColumn: 'Hide',
+  columnMenuUnsort: 'Unsort',
+  columnMenuSortAsc: 'Sort by Asc',
+  columnMenuSortDesc: 'Sort by Desc',
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: (count) => `${count} active filter(s)`,
+  columnHeaderFiltersLabel: 'Show Filters',
+  columnHeaderSortIconLabel: 'Sort',
+
+  // Rows selected footer text
+  footerRowSelected: 'row selected',
+  footerRowSelectedPlural: (count) =>
+    count > 1
```

Comment:
Zero should be plural from what I understand cc @mbrookes 

```suggestion
    count !== 1
```

Reply:
If it is zero then that label is not shown. This is the current functionality.

Reply:
OK, so the default is good.

Can this display logic be customized by a developer?

Reply:
I actually updated it with your suggestion so it is be good now